### PR TITLE
FML support

### PR DESCRIPTION
**Pull request by dotStart**
Recent builds of the FML (Forge Mod Loader) seem to crash due to missing information about the custom launcher. Java couldn't find net.minecraft.Launcher (or something like that) which is a missing reference to the official launcher.

### CODE REVIEW

**Comment by sk89q** [comment]
Would you be able to fix the wonky indentation? (No tabs, 4 spaces)

**Comment by dotStart** [comment]
Fixed. Sorry :-)

File: src/main/java/com/sk89q/mclauncher/launch/GameAppletContainer.java [hunk 1]
```diff
@@ -97,6 +97,22 @@ public String getParameter(String name) {
         return null;
     }
 
+    /**
+     * Allows to replace the wrapped applet in runtime.
+     * @param			applet
```
Comment (md-5) [inline src/main/java/com/sk89q/mclauncher/launch/GameAppletContainer.java]: Pretty sure thats not the javadoc style we use.

File: src/main/java/com/sk89q/mclauncher/launch/GameAppletContainer.java [hunk 2]
```diff
@@ -97,6 +97,18 @@ public String getParameter(String name) {
         return null;
     }
 
+    public void replace(Applet applet) {
+    	// stop old applet
```
Comment (md-5) [inline src/main/java/com/sk89q/mclauncher/launch/GameAppletContainer.java]: What he originally meant is these // are 1 line too far to the left. You can also add the javadoc back, one space between @ param and then name of the variable.

**Comment by dotStart** [comment]
I hope this is correct now (9001 commits for one single change xD)

**Comment by md-5** [comment]
Sure, I'll do a manual merge so it lands up as one tidy package with your name. Tomorrow though.

**Comment by md-5** [comment]
Merged

**Pull request closed without merging**